src/includes/macros/developers/cards.njk

```diff
@@ -0,0 +1,30 @@
+{% macro card(content) %}
+<div class="pt-6">
+    <div class="flow-root bg-gray-100 rounded-lg px-6 pb-8">
+    <div class="-mt-6">
+        <div>
```

Comment:
Might be wrong, but I think that `<div>` wrapping the `<span>` can be safely removed